### ISSUE TITLE
NTR: Contact lens wearing (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7998]
New term request Contact lens wearing (segal_211201185252):   UMLS: 0920138

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000098
- label: Contact lens wearer
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:26292204
- def: A past medical history of wearing contact lenses.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7998
- synonyms:
